packages/block-editor/src/components/button-block-appender/style.scss

```diff
@@ -3,12 +3,16 @@
 	flex-direction: column;
 	align-items: center;
 	justify-content: center;
-	padding: $grid-unit-10;
 	width: 100%;
 	height: auto;
 	color: $gray-900;
 	box-shadow: inset 0 0 0 $border-width $gray-900;
 
+	// Needs specificity to override button styles.
+	&.components-button.components-button {
+		padding: $grid-unit-15;
+	}
```

Comment:
Note that this component impacts more than just the "group" block. It can be used by third-party blocks... Just want to make sure the change is safe for everyone.

Reply:
The problem right now is that the padding registered for this button doesn't take at all, because it's overridden by `.edit-post-visual-editor .components-button`:
<img width="440" alt="Screenshot 2022-03-31 at 14 41 19" src="https://user-images.githubusercontent.com/1204802/161057038-c42b5988-2eb8-4530-a776-e9186c357398.png">

So this change does increase the padding to `12px`, but shipping in trunk it's currently `6px 12px` because of the inheritance.